bottlecap/src/secrets/decrypt.rs

```diff
@@ -43,7 +40,7 @@ pub async fn resolve_secrets(config: Arc<Config>, aws_config: &AwsConfig) -> Opt
             }
         }
     } else {
-        return None;
+        Some(config.api_key.clone())
```

Comment:
Will this resolve to None since the API key wouldn't be set either?

Reply:
Not sure what you mean.
Why shouldn't it be set? And how would it resolve to None?
I moved the cleanup code inside, so maybe it's less confusing

Reply:
If it's empty, we should send None